cookbook/getSortedLocalDateTimes.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * getSortedLocalDateTimes will sort an array of zoneless Temporal.DateTime instances by the
+ * corresponding local date and time of day (e.g., for building a conference schedule).
+ *
+ *
+ * @param {Temporal.DateTime[]} dateTimes This is a Date instance
+ * @param {boolean} direction accending or descending order
+ * @returns {Temporal.Absolute} Temporal.Absolute instance
+ */
+function getSortedLocalDateTimes(dateTimes, reverse = false) {
+    let newDateTimes = dateTimes.sort((a, b) => {
+        return a.compare(b);
+    });
+
+    return reverse ? newDateTimes.reverse() : newDateTimes;
+}
+
+let a = new Temporal.DateTime({ day: 4, month: 2 });
```

Comment:
These would need to be `Temporal.DateTime.from({...})` instead of the constructor (which takes all the units as positional arguments).

Currently in the polyfill `from({ day: 4, month: 2 })` would be legal and result in midnight, February 4, 1 BCE; in my opinion it might be better to use more current dates, perhaps from an actual conference schedule since you mentioned it above in the docs.